cdn/cdn_generator/common_files.py

```diff
@@ -0,0 +1,14 @@
+import pathlib
+import shutil
+
+
+def generate_common_files():
+    pathlib.Path("generated").mkdir(parents=True, exist_ok=True)
+    shutil.copyfile("config.yaml", "generated/config.yaml")
+    shutil.copyfile("README.md", "generated/README.md")
+
+    pathlib.Path("generated/errors").mkdir(parents=True, exist_ok=True)
+    shutil.copyfile("errors/404.html", "generated/errors/404.html")
```

Comment:
maybe a comment explaining why these nongenerated files need to be in the generated folder?